src/main/java/de/chojo/repbot/config/elements/Botlist.java

```diff
@@ -5,6 +5,8 @@
 @SuppressWarnings({"FieldMayBeFinal", "FieldCanBeLocal"})
 public class Botlist {
     private boolean submit;
+    private String host = "0.0.0.0";
+    private int port = 8888;
     private BotListConfig topGg = new BotListConfig("top.gg", "", "", 264445053596991498L, "", "");
     private BotListConfig discordBotsGg = new BotListConfig("discord.bots.gg", "", "", 110373943822540800L, "", "");
     private BotListConfig discordBotlistCom = new BotListConfig("discordbotlist.com", "", "", 450100127256936458L, "", "");
```

Comment:
The empty strings, should they be set via configuration? I don't know how Jackson works, but does it allow pre setting some variables which will be overwritten on configuration load?